tests/tests_pytorch/profilers/test_hpu_profiler.py

```diff
@@ -0,0 +1,224 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import glob
+import json
+import os
+import shutil
+
+import pytest
+
+from pytorch_lightning import Trainer
+from pytorch_lightning.accelerators import HPUAccelerator
+from pytorch_lightning.demos.boring_classes import BoringModel
+from pytorch_lightning.profilers import AdvancedProfiler, HPUProfiler, SimpleProfiler
+from tests_pytorch.helpers.runif import RunIf
+
+
+class TestHPUProfiler:
+    def setup_method(self):
+        shutil.rmtree("profiler_logs", ignore_errors=True)
+
+    def teardown_method(self):
+        shutil.rmtree("profiler_logs", ignore_errors=True)
```

Comment:
Can we rather use plain function as everywhere else and, instead of this cleaning use temp folders?

Reply:
yeah, I agree we should use plain functions here. the logs are in the tmp folder.

Reply:
@jerome-habana yup, let's remove it.

Reply:
Removing this will have issues with multi-process runs. There is no single unified directory to hold the logs